packages/core/src/baseclient.ts

```diff
@@ -247,6 +247,7 @@ export abstract class BaseClient<B extends Backend, O extends Options> implement
    * @param scope A scope containing event metadata.
    * @returns A new event with more information.
    */
+  // tslint:disable-next-line:cyclomatic-complexity
```

Comment:
😅 

Maybe in order to remove this, you could refactor those lines into a private function
https://github.com/getsentry/sentry-javascript/pull/2627/files/d61574adc0829957ecc4790aa4c557a56deefff1#diff-377e14ef259b8bd726753063b8658e97R256-R288